cime_config/testdefs/testmods_dirs/clm/transientERI/user_nl_clm

```diff
@@ -0,0 +1,5 @@
+
+! The following is needed in a transient ERI test so that the hybrid case works,
+! given the 2-year jump from the initial case to the hybrid case:
+! https://github.com/ESMCI/cime/blob/f0b9d218e97eb92ea84decaf6ae1bfc263b8ddc9/CIME/SystemTests/eri.py#L93
```

Comment:
I find the referencing of the specific hash in cime a bit confusing here. I'm not sure that the code reference is helpful, just the description is probably fine.

It does make me wonder if this should be the default for a hybrid case though? I'm siding for it not to be, but I see reasons to make it the default for hybrid.